docs/source/distributed_systems.rst

```diff
@@ -59,6 +59,26 @@ address by its alias:
 As you can see, aliases become specially useful in distributed systems!
 
 
+.. _shutdown_nameserver:
+
+Shutting down name servers
+--------------------------
+
+We can finish the execution of a name server through the
+:func:`nameserver.shutdown <osbrain.nameserver.shutdown>` method.
+
+Shutting down a name server will also shut down any registered agents on it.
+This is specially useful when within our system, we are going to have
+various independent groups of agents. That way, we can register all agents of
+each group in its own name server and easily shut down each group by a
+:func:`nameserver.shutdown <osbrain.nameserver.shutdown>` call to its
+name server.
+
+Note that even if a name server is running without any registered agents,
+it will be kept alive until a call to
+:func:`nameserver.shutdown <osbrain.nameserver.shutdown>`.
```

Comment:
This last paragraph seems superfluous to me, but if you think it could help someone, keep it there.